CONTRIBUTING.md

```diff
@@ -1,39 +1,167 @@
+    - Any configuration you've made
+  - What did you expect to happen?
+  - What happened? Include as much information as possible.
+    - Nature of reported defect (e.g. user name missing, not "It doesn't work."). Is it intermittent?
+    - The best help here is a failing test.
+    - Then the steps to reproduce and/or a gist or repository that demonstrates the defect.
+    - Then examples of the code you were using.
+    - Any error messages (including stacktrace, i.e. ""Show me the error.")
+  - Things you've tried.
+  - A pull request for your fix would be great.  Code should have tests.
+  - Link to source code, if available
+
+Please make sure only to include one issue per report.
+If you encounter multiple, unrelated issues, please report them as such.
+
+Simon Tatham has written an excellent on article on
+[How to Report Bugs Effectively](http://www.chiark.greenend.org.uk/~sgtatham/bugs.html)
+which is well worth reading, although it is not specific to ActiveModelSerializers.
+
+
 
-The first place to start is by looking at our [GitHub Issues](https://github.com/rails-api/active_model_serializers/issues).
+1. What have you tried?
 
-The vast majority of development is happening under the `master` branch, currently slated for release as `0.10.x`. This is where we would suggest you start.
+Include as much sample code as you can to help us reproduce the issue. (Inline, repo link, or gist, are fine. A failing test would help the most.)
 
-Fixing bugs is extraordinarily helpful and requires the least familiarity with AMS. Look for issues labeled [**Needs Bug Verification**](https://github.com/rails-api/active_model_serializers/labels/Needs%20Bug%20Verification) and [**Bug**](https://github.com/rails-api/active_model_serializers/labels/bug).
+This is extremely important for narrowing down the cause of your problem.
 
-We are also actively working to identify tasks under the label [**Good for New Contributors**](https://github.com/rails-api/active_model_serializers/labels/Good%20for%20New%20Contributors). Some bugs are expressly not good for new contributors, so don't expect 100% overlap between the two.
+Thanks!
 
-If you want to work on new feature development, look for the label [**Feature**](https://github.com/rails-api/active_model_serializers/labels/Feature).
+Sometimes an issue will be closed by a maintainer for various reasons.  In some cases, this is
+an invitation to make a better case for your issue or be able to reproduce a bug, and
+its being close is just an opportunity to help out some more, and then re-open.
 
-We are also encouraging comments to substantial changes (larger than bugfixes and simple features) under an "RFC" (Request for Comments) process before we start active development. Look for the [**RFC**](https://github.com/rails-api/active_model_serializers/labels/RFC) label.
+#### After
+
+Thanks to everyone involved!
+
+If you get help, sharing it back in the form of a pull-request or making an issue to document
```

Comment:
"PRs, when including new behaviors should include whenever possible a new suitable recommendation about how to use the feature, you can find examples at `/docs`"